app/services/concerns/xccdf/profile_rules.rb

```diff
@@ -7,26 +7,29 @@ module ProfileRules
 
     included do
       def save_profile_rules
-        ::ProfileRule.transaction do
```

Comment:
I feel like this is no longer necessary after https://github.com/RedHatInsights/compliance-backend/pull/1150 cc @vkrizan